fleetspeak/src/client/internal/monitoring/resource_usage_fetcher.go

```diff
@@ -39,6 +39,12 @@ type ResourceUsage struct {
 
 	// Resident set size for a process, in bytes.
 	ResidentMemory int64
+
+	// Client IO data read so far, in bytes.
+	IORead int64
```

Comment:
I would use the names `IOReadBytes` and `IOWriteBytes` to make sure the measurement unit is clearly indicated.